lib/http_clients/creditas/person_deletion_api.ex

```diff
@@ -1,7 +1,24 @@
 defmodule HttpClients.Creditas.PersonDeletionApi do
   @moduledoc false
 
-  alias HttpClients.Creditas.PersonDeletionApi.PersonDeletion
+  alias HttpClients.Creditas.PersonDeletionApi.{Acknowledgment, PersonDeletion}
+
+  @spec acknowledgments(Tesla.Client.t(), Acknowledgment.t()) :: {:error, any} | :ok
+  def acknowledgments(client, ack) do
```

Comment:
achei um pouco estranho chamar de aacknowledgements essa função, pensei em algo tipo
```suggestion
  def acknowledge_deletion(client, ack) do
```
ou
```suggestion
  def ack_deletion(client, ack) do
```
oq acha?